strawberry/schema/schema_converter.py

```diff
@@ -33,6 +34,13 @@
 from .types.scalar import get_scalar_type
 
 
+class CustomGraphQLEnumType(GraphQLEnumType):
```

Comment:
Clean solution 👍

Should we add a comment documenting why this custom class was added? The old code had the following comment:

```python
# graphql-core expects a resolver for an Enum type to return
# the enum's *value* (not its name or an instance of the enum).
```

Reply:
Good point, will add.